models/events/dependency/DependencyType.js

```diff
@@ -1,5 +1,6 @@
 module.exports = require('../../Enum').create({
   values: [
-    'NPM'
+    'NPM',
+    'PyPi'
```

Comment:
Please make this an uppercase to be consistent. It should be referenced like a constant:

```js
DependencyType.NPM
DependencyType.PYPI
```